### ISSUE TITLE
Contenteditable property missing on safari

### ISSUE THREAD

**Issue by KevenImbeault** [tinymce/tinymce#8717]
**What is the current behavior? Describe the bug**
Safari browser is unable to input text inside the editor, because the contenteditable="true" property is missing.
<img width="1218" alt="image" src="https://github.com/tinymce/tinymce/assets/17691536/8bf9e6f4-f709-45e8-bc19-2fa3c87d4738">

 Other browsers such as Firefox do have that tag and are usable
<img width="1102" alt="image" src="https://github.com/tinymce/tinymce/assets/17691536/8e1801d1-44ae-41f8-ba2d-ba00b6691273">

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**

**What is the expected behavior?**
The user should be able to input text inside the editor like it is possible for other browsers

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
Using Safari browser on MAC OS (Windows version is not supported anymore), according to the official website examples the version 5 editor works on safari

**Comment by TheSpyder** [comment]
As far as we know version 5 does still work on Safari.
![Screenshot 2023-05-11 at 9 33 49 am](https://github.com/tinymce/tinymce/assets/298292/f5e51e19-19d7-4184-947c-c1e9d778f344)

Some questions:
- what version of Safari are you using?
- perhaps there's a Safari extension blocking this?
- are there any exceptions in the console?

**Comment by KevenImbeault** [comment]
Yes version 5 does work, but I'm using v6. Sorry looks like I forgot to mention it.

Safari has no extensions and no exceptions gets thrown in the console.

**Comment by KevenImbeault** [comment]
Validated today, an extension was apparently in use (Zotero). V6 now works so I'm closing this
